src/api/dav-import.js

```diff
@@ -0,0 +1,20 @@
+const BASE_PATH = '/import';
+
+export default function(client) {
```

Comment:
where is this `client` parameter injected?

Reply:
Please check the readme for the usage documentation ;)

Reply:
ok, got it :D